tests/foreman/ui/test_oscapcontent.py

```diff
@@ -0,0 +1,95 @@
+    """Implements Oscap Content tests in UI."""
+
+    @classmethod
+    def setUpClass(cls):
+        super(OpenScapContent, cls).setUpClass()
+        cls.content_path = get_data_file(OSCAP_CONTENT_FILE)
+
+    def test_positive_create_oscapcontent(self):
+        """@Test: Create OpenScap content.
+
+        @Feature: OpenScap - Positive Create.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+
+        @Assert: Whether creating  content for OpenScap is successful.
+
+        """
+        with Session(self.browser) as session:
+            for content_name in valid_data_list():
+                with self.subTest(content_name):
+                    set_context(session, org=ANY_CONTEXT['org'])
+                    session.nav.go_to_oscap_content()
+                    make_oscapcontent(
+                        session,
+                        name=content_name,
+                        content_path=self.content_path,
+                    )
+                    self.assertIsNotNone(
+                        self.oscapcontent.search(content_name))
+
+    def test_positive_default_oscapcontent(self):
+        """@Test: Check whether OpenScap content exists by default.
+
+        @Feature: OpenScap - Test Default Content.
+
+        @Steps:
+
+        1. Navigate to oscap Content page.
+        2. Set Org as Any Org.
+
+        @Assert: Whether oscap content exists by default.
+
+        """
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            session.nav.go_to_oscap_content()
+            self.assertIsNotNone(self.oscapcontent.search(
+                OSCAP_DEFAULT_CONTENT['rhel7_content']))
+            self.assertIsNotNone(self.oscapcontent.search(
+                OSCAP_DEFAULT_CONTENT['rhel6_content']))
+
+    def test_positive_delete_oscapcontent(self):
+        """@Test: Create OpenScap content and then delete it.
+
+        @Feature: OpenScap - Delete.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+        3. Delete the openscap content.
+
+        @Assert: Deleting content for OpenScap is successful.
+
+        """
+        with Session(self.browser) as session:
+            for content_name in valid_data_list():
+                with self.subTest(content_name):
+                    set_context(session, org=ANY_CONTEXT['org'])
+                    session.nav.go_to_oscap_content()
+                    make_oscapcontent(
+                        session,
+                        name=content_name,
+                        content_path=self.content_path,
+                    )
+                    self.assertIsNotNone(
+                        self.oscapcontent.search(content_name))
+                    self.oscapcontent.delete(content_name)
+                    self.assertIsNone(self.oscapcontent.search(content_name))
```

Comment:
oh! yeah nice that this is implicit now :+1: , missed to remove it, this code was lying since a long time , while I had been working on other stuff.